### PR TITLE
Update Tutorial Implementation

### PR DESCRIPTION
**Pull request by maj0-0**

### CODE REVIEW

**Review by PearlynnT — COMMENTED** [review]
Can migrate Tutorial.java to a separate folder like SocialMediaLink for consistency

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 1]
```diff
@@ -235,15 +237,15 @@ public void parseTutorials_validInput_success() throws Exception {
...
 
-        // Replace the following with actual expected values if needed
         EditPersonDescriptor expectedDescriptor = new EditPersonDescriptorBuilder()
-            .withTutorials(List.of(new Tutorial("01"), new Tutorial("02"), new Tutorial("03")))
+            .withTutorials(new HashSet<>(List.of(new Tutorial("01"), new Tutorial("02"), new Tutorial("03"))))
             .build();
 
         assertEquals(expectedDescriptor, descriptorBuilder.build());
     }
+
```
Comment (wnchan) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:250]: LGTM! but i think can add invalid tutorial value test cases to increase the code coverage.

**Review by wnchan — COMMENTED** [review]
LGTM! Maybe can add more test cases to increase code coverage.

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 2]
```diff
@@ -50,9 +52,9 @@
 import seedu.address.model.person.Email;
 import seedu.address.model.person.Major;
 import seedu.address.model.person.Name;
-import seedu.address.model.person.Tutorial;
 import seedu.address.model.person.Year;
 import seedu.address.model.socialmedialink.SocialMediaLink;
+import seedu.address.model.tutorial.Tutorial;
 import seedu.address.testutil.EditPersonDescriptorBuilder;
 
 public class EditCommandParserTest {
```
Comment (PearlynnT) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:60]: update the method `parse_invalidValue_failure` with invalid tutorial, update `parse_allFieldsSpecified_success`, update `parse_oneFieldSpecified_sucesss`

**Pull request merged**